db-engine/src/main/java/com/splicemachine/db/impl/jdbc/EmbedConnection.java

```diff
@@ -101,7 +102,8 @@
  */
 public abstract class EmbedConnection implements EngineConnection
 {
-	protected static final StandardException exceptionClose = StandardException.closeException();
+    public static final ThreadLocal<Boolean> isCreate = new ThreadLocal<>();
```

Comment:
is it used somewhere in subclasses? in EmbedConnection I see only .set and .remove

Reply:
It is used in `SpliceDatabase.bootStore`
It is quite similar to `EngineLifecycleService.isCreate` that we use when creating the very first `splicedb`. However, when we create a new database, we don't go all the way to `EngineLifecycleService.start()` so we cannot use that one class variable.